layouts/community/ortho_4x12/bredfield/keymap.c

```diff
@@ -0,0 +1,181 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Keymap layers
+#define _BASE_LAYER 0
+#define _RAISE_LAYER 1
+#define _LOWER_LAYER 2
+#define _NUMPAD_LAYER 3
+#define _NAVIGATION_LAYER 4
+#define _UI_LAYER 5
+#define _KEYBOARD_LAYER 6
+
+// Key aliases
```

Comment:
QMK defines aliases for these keycodes automatically.

https://github.com/qmk/qmk_firmware/blob/53a81fc2f681eda7267804eb990bc24c4ef58512/quantum/quantum_keycodes.h#L29-L31

If you use the predefined aliases in your keymap, you don't need to define either of these.